fastlane/lib/fastlane/actions/update_info_plist.rb

```diff
@@ -34,18 +34,15 @@ def self.run(params)
           # Read existing plist file
           info_plist_path = File.join(folder, "..", params[:plist_path])
           UI.user_error!("Couldn't find info plist file at path '#{info_plist_path}'") unless File.exist?(info_plist_path)
-          plist = Xcodeproj::Plist.read_from_path(info_plist_path)
 
-          # Update plist values
-          plist['CFBundleIdentifier'] = params[:app_identifier] if params[:app_identifier]
-          plist['CFBundleDisplayName'] = params[:display_name] if params[:display_name]
-          params[:block].call(plist) if params[:block]
-
-          # Write changes to file
-          Xcodeproj::Plist.write_to_path(plist, info_plist_path)
-
-          UI.success("Updated #{params[:plist_path]} 💾.")
-          File.read(info_plist_path)
+          UpdatePlistAction.run(
+            plist_path: info_plist_path,
+            block: lambda { |plist|
```

Comment:
Can we change this to use `proc do`? This is how other actions like `pem` do this so just for consistency purposes 😊 
```rb
block: proc do |plist|
  plist['CFBundleIdentifier'] = params[:app_identifier] if params[:app_identifier]
  plist['CFBundleDisplayName'] = params[:display_name] if params[:display_name]
  params[:block].call(plist) if params[:block]
end
```